### PR TITLE
Update documentation and remove obsolete transformations

### PR DESCRIPTION
**Pull request by VictorCMiraldo**
* removes `destrNF` and `pullDestr` (and dependencies)
* update documentation and tutorial to reflect the new command line options

### CODE REVIEW

File: tests/integration/MultiSigStateMachine/TUTORIAL.md [hunk 1]
```diff
@@ -141,90 +141,32 @@ write the equivalent command line for your contract.
...
-           @(Maybe (Tuple20 (TxConstraints0 Void Void) (State0 MSState)))
-           (λ (pmt1107 : Payment0) (pks1108 : List0 ByteString)
-            . Input_match 4#i1100
-                @(Maybe (Tuple20 (TxConstraints0 Void Void) (State0 MSState)))
-  ...
-```
-
-This is a simplification of the PlutusIR representation of the
-`transition` function and is what pirouette is going to use
-when extracting the TLA+ model from it.  Yet, the first
```
Comment (GuillaumeGen) [inline tests/integration/MultiSigStateMachine/TUTORIAL.md:187]: I think that from line 157 to this point should be kept, either at the end of this file or in a file called `Internal.md` referenced in the read me.

File: README.md [hunk 2]
```diff
@@ -71,6 +71,10 @@ saveBinaryFile = case getPir $$(PlutusTx.compile [|| mkValidator ||]) of
                    Nothing  -> error "plutus compilation failed"
 ```
 
+Note that you _do not_ need to import Pirouette to get a `.pir` or a `.flat` file from your contract,
+The `getPir` function comes from `plutus-core`.
+
```
Comment (GuillaumeGen) [inline README.md:76]: Should we add an example `savePirFile` function?

**Review by GuillaumeGen — APPROVED** [review]
I love the `README` for manual tests. Really a great idea

File: tests/integration/MultiSigStateMachine/TUTORIAL.md (hunk 1 above)
Comment (VictorCMiraldo) [reply tests/integration/MultiSigStateMachine/TUTORIAL.md:187]: Makes sense! I'll keep it and move it to the end of the file

File: README.md (hunk 2 above)
Comment (VictorCMiraldo) [reply README.md:76]: Why not?! I'll add it

**Pull request merged**